### PR TITLE
kokoro build files

### PR DESCRIPTION
**Pull request by akerekes**

### CODE REVIEW

File: build/build.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```
Comment (loosebazooka) [inline build/build.sh]: Just wondering:
build is a directory that gradle generates also and we tend to ignore it, is this a convention or can it be anything?

File: kokoro/ubuntu/continuous.cfg [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# Format: //devtools/kokoro/config/proto/build.proto
+
+# Location of the continuous bash script in repository.
+build_file: "appengine-plugins-core/kokoro/ubuntu/continuous.sh"
```
Comment (loosebazooka) [inline kokoro/ubuntu/continuous.cfg]: So for windows/mac we just add in more buildfiles here?

File: build/build.sh (hunk 1 above)
Comment (loosebazooka) [inline build/build.sh]: I mean this repo is maven build, but if we transition others.

File: kokoro/ubuntu/continuous.sh [hunk 3]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```
Comment (loosebazooka) [inline kokoro/ubuntu/continuous.sh]: What's the difference between build and continuous?

File: build/build.sh (hunk 1 above)
Comment (akerekes) [inline build/build.sh]: We can call this directory something else than `build`, e.g. `build-scripts`?

File: kokoro/ubuntu/continuous.cfg (hunk 2 above)
Comment (akerekes) [inline kokoro/ubuntu/continuous.cfg]: We'll create separate directories for Win and Mac under the `kokoro` directory and they'll refer their respective files.

File: kokoro/ubuntu/continuous.sh (hunk 3 above)
Comment (akerekes) [inline kokoro/ubuntu/continuous.sh]: Actually we can get rid of `build.sh` and add the same logic into `continuous.sh`, and move `continuous.sh` up to the `kokoro` directory (and share it between Ubuntu and Mac)

File: kokoro/ubuntu/continuous.cfg [hunk 4]
```diff
@@ -0,0 +1,4 @@
+# Format: //devtools/kokoro/config/proto/build.proto
+
+# Location of the continuous bash script in repository.
```
Comment (elharo) [inline kokoro/ubuntu/continuous.cfg]: continuous build bash

**Pull request merged**